### PR TITLE
Update websockets

### PR DESCRIPTION
**Pull request by csarven**

### CODE REVIEW

**Review by kjetilk — APPROVED** [review]
@csarven and I wordsmithed this together, so I am already in agreement.

File: protocol.html [hunk 1]
```diff
@@ -886,19 +886,13 @@ <h2 property="schema:name">Notifications</h2>
...
-                </div>
-              </section>
+              <p>For real-time communication between client and server about changes affecting a resource, the Solid Protocol uses the WebSocket API [<cite><a class="bibref" href="#bib-w3c-html">W3C-HTML</a></cite>] and the WebSocket Protocol.</p>
 
-              <section id="websockets-patching" inlist="" rel="schema:hasPart" resource="#websockets-patching">
-                <h3 property="schema:name">WebSockets Patching</h3>
-                <div datatype="rdf:HTML" property="schema:description">
-                </div>
-              </section>
+              <p>The <cite><a href="https://github.com/solid/solid-spec/blob/master/api-websockets.md" rel="cito:citesAsPotentialSolution">Solid WebSockets API</a></cite> Unofficial Draft is implemented by some implementations. This draft does not include an authentication mechanism, and is therefore inappropriate for future use.</p>
```
[the remark this answers is not in the document]
Comment (justinwb) [reply protocol.html:893]: A bit of nitpicking but expanding on @acoburn's suggestion it may be better to say 'implemented some clients and servers'

**Review by justinwb — APPROVED** [review]
One minor suggestion extending aaron's but overall looks good ✅

**Review by RubenVerborgh — APPROVED** [review]
Excellent.

**Review by kjetilk — APPROVED** [review]
Good to go as it has @timbl 's comments incorporated.

**Pull request merged**